### PR TITLE
feat: added rules component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by Sahil25061999 — APPROVED** [review]
didnt find any issues, love the minimalism of the UI. Just a suggestion the search container can be made a bit dark.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    <h4>Click the button below to dicover the categories of quiz</h4>
+                    <a href="screens/rules.html" class="btn-small margin-lr">Take the quiz</a>
+                </div>
+                <div class="quiz-home-rt">
+                    <img class="quiz-image" src="img/undraw_quiz_nlyh.svg" alt="quiz-image">
+                </div>
+            </div>
+
+
+        </div>
```
Comment (Ankur9669) [inline index.html:46]: fix: remove extra spacing here

File: screens/rules.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
+                <img class="pointer" src="https://img.icons8.com/material/36/000000/user-male-circle--v1.png" />
+
+            </div>
```
Comment (Ankur9669) [inline screens/rules.html:26]: fix: remove extra spacing here

File: screens/rules.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                    <div class="rules-item">
+                        <h4>1. Language:</h4>
+                        <p>The quizzes are all in English, but you may answer in any official language of a country with a team participating in
+                        the Nations Team Quiz at the Quiz Olympiad.</p>
+                        <h4>2. Challenges:</h4>
+                        <p>If at any time you think your answer should be accepted, then you must complete a challenge slip.</p>
+                        <h4>3. Cheating:</h4>
+                        <p>Anyone suspected by an official of cheating will be disqualified. You must not use any mobile device during a quiz
+                        competition.</p>
+                    </div>
```
Comment (Ankur9669) [inline screens/rules.html:56]: Suggestion: These are li elements

File: screens/rules.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    <a href="#" class="btn-small margin-lr">Start</a>
+                </div>
+            </div>
+
+
+        </div>
```
Comment (Ankur9669) [inline screens/rules.html:60]: fix: remove unused spacing

File: style.css [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+}
+
+*{
+    font-family: 'Montserrat', sans-serif;
+}
+
+body{
+    margin: 0;
+}
```
Comment (Ankur9669) [inline style.css:28]: Suggestion: you can also add padding: 0 here

**Review by Ankur9669 — APPROVED** [review]
The code looks good, I am not sure if you have made it responsive or not.

But it is good to merge

File: index.html (hunk 1 above)
Comment (eakansh125) [reply index.html:46]: sure

File: screens/rules.html (hunk 2 above)
Comment (eakansh125) [reply screens/rules.html:26]: sure

File: screens/rules.html (hunk 3 above)
Comment (eakansh125) [reply screens/rules.html:56]: okay

**Comment by eakansh125** [comment]
> didnt find any issues, love the minimalism of the UI. Just a suggestion the search container can be made a bit dark.

Will work on that, thanks

**Pull request merged**